packages/fxa-auth-server/lib/sentry.js

```diff
@@ -131,6 +129,33 @@ async function configureSentry(server, config) {
             return sentryEvent;
           });
           scope.setExtra('exception', exception);
+          const cause = verror.cause(err);
```

Comment:
Hmm, I find it somewhat distressing that the unit tests for the sentry module are somewhat non-existent: https://github.com/mozilla/fxa/blob/d8b7629291ec6e0c3846c6c8954521cffd145884/packages/fxa-auth-server/test/local/sentry.js#L7